frontend/src/metabase/query_builder/components/ExpressionPopover.jsx

```diff
@@ -56,14 +57,17 @@ export default class ExpressionPopover extends React.Component {
                 onDone(expression);
               }
             }}
+            onBlankChange={newBlank => {
+              this.setState({ isBlank: newBlank });
+            }}
```

Comment:
This seems oddly specific. Is it because we need a change event that fires before blur?

Reply:
so two alternatives: change button state based upon not being blank and change button state based upon non erroring. but it felt to me very weird to have the button disable and enable as you type, because that flashing gets your attention, which is why I wanted it to just enable and just stay enabled when typing. you still get the errors when you actually blur